lib/note-info.jsx

```diff
@@ -1,11 +1,15 @@
 import React, { PropTypes } from 'react';
-import { includes } from 'lodash';
+import { includes, isEmpty } from 'lodash';
 import ToggleControl from './controls/toggle';
 import moment from 'moment';
 import CrossIcon from './icons/cross';
-import { isEmpty } from 'lodash';
+import { connect } from 'react-redux';
+import { actionCreators } from './flux/app-state';
+import { setMarkdown } from './state/settings/actions';
+//import { tracks } from './analytics'
```

Comment:
Dang, I put this here at some point because I was sure I saw it used in some of the imported functions, but eventually I was wrong and forgot about it. :/